oneanddone/tasks/templates/tasks/detail.html

```diff
@@ -164,5 +181,32 @@
       </div>
     {% endif %}
   </div>
+
+  {% if attempt and task.must_be_verified %}
+    {% if attempt.communication_set.exists() %}
+      <div class="row">
+        <fieldset class="verify-task-fieldset">
+          <legend>{{ _('Verification History') }}</legend>
+          {{ communication_list(attempt.communication_set.all()) }}
+        </fieldset>
+      </div>
+    {% endif %}
+    <div class="row">
+      <div>
+        <fieldset class="verify-task-fieldset">
+          <legend>{{ _('Task Verification') }}</legend>
+          <div>
+            {{ _('Administrators will verify your work to complete this task.
+                Follow the instructions and click the button to submit your work.') }}
```

Comment:
Use `{% trans %}` blocks for multiline strings. Otherwise I believe tower will extract a string with many whitespaces between `this task.` and `Follow`